ingestion/setup.py

```diff
@@ -324,6 +324,7 @@
     "minio==7.2.5",
     *plugins["mlflow"],
     *plugins["datalake-s3"],
+    "requests==2.31.0",
```

Comment:
I think we should be good to revert this https://github.com/open-metadata/OpenMetadata/pull/16365/files#diff-51c0c123b825b1c331fadd475a97dd0783282959ae7822a4b3993fe7dd09ab8dR111

Reply:
I removed the pinning on base and kept this one